src/index.js

```diff
@@ -0,0 +1,174 @@
+                await db.query(query);
+                console.log(`Created ${firstName} ${lastName} as an employee!`);
+            } else {
+                console.warn("[ERROR]: Please enter a role before proceeding...");
+            }
+        }
+        if (option === "viewEmployee") {
+            const query =
+                'SELECT CONCAT(employee_role.first_name, " ", employee_role.last_name) AS Employee, title as "Role", salary as "Salary",  name as "Department",  CONCAT (employee_manager.first_name, " ", employee_manager.last_name) as "Manager"  FROM employee employee_role LEFT JOIN role ON employee_role.role_id = role.id LEFT JOIN department ON role.department_id = department.id LEFT JOIN employee employee_manager ON employee_role.manager_id = employee_manager.id;';
+            const data = await db.query(query);
+            console.table(data);
+        }
+        if (option === "updateEmployeeRole") {
+            const employee = await selectAllFromEmployeeTable(db);
+
+            if (employee.length) {
+                const questions = await updateRoleQuestions(db);
+                const { employees, role_id } = await inquirer.prompt(questions);
+                const query = `UPDATE employee SET role_id = ${role_id} WHERE id = ${employees};`;
+                await db.query(query);
+            }
+        }
+        if (option === "updateEmployeeManager") {
+            const employee = await selectAllFromEmployeeTable(db);
+
+            if (employee.length) {
+                const questions = await updateManagerQuestions(db);
+                const { employee, manager } = await inquirer.prompt(questions);
+
+                const query = `UPDATE employee SET manager_id = ${manager} WHERE id = ${employee};`;
+                await db.query(query);
+            }
+        }
+        if (option === "viewEmployeeDepo") {
+            const questions = await viewDepartmentQuestion(db);
+            const { department } = await inquirer.prompt(questions);
+            const query = `SELECT first_name, last_name FROM employee INNER JOIN role ON employee.role_id = role.id WHERE department_id = ${department};`;
+            console.table(await db.query(query));
+        }
+
+        // view managers
+        if (option === "viewManager") {
+            const managerQuery =
+                "SELECT first_name AS firstName, last_name AS lastName FROM employee WHERE manager_id IS NULL;";
+            const data = await db.query(managerQuery);
+            console.table(data);
+        }
+
+        // delete options
+        if (option === "deleteDepartment") {
+            const questions = await deleteDepartmentQuestion(db);
+            const { department } = await inquirer.prompt(questions);
+            const query = `DELETE FROM department WHERE id = ${department};`;
+            await db.query(query);
+        }
```

Comment:
Please keep spacing consistent and add a spare line here and after each `if` block